subsys/bluetooth/adv_prov/providers/Kconfig.fast_pair

```diff
@@ -36,7 +36,7 @@ config BT_ADV_PROV_FAST_PAIR_SHOW_UI_PAIRING
 
 choice BT_ADV_PROV_FAST_PAIR_BATTERY_DATA_MODE
 	prompt "Select advertising battery data mode"
-	default BT_ADV_PROV_FAST_PAIR_BATTERY_DATA_SHOW_UI
+	default BT_ADV_PROV_FAST_PAIR_BATTERY_DATA_NONE
```

Comment:
Do we need to override the Bluetooth Fast Pair sample configuration or do we select the mode explicitly in its prj.conf file?

Reply:
In sample we set the battery mode at runtime using `bt_le_adv_prov_fast_pair_set_battery_mode` so the default value doesn't really matter in that case.